Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN apk add --virtual build-deps gcc python3-dev openldap-dev musl-dev git
 # and we need to keep the .git directory intact.
 # The workaround is to copy everything, but limit it with .dockerignore.
 COPY . .
-COPY --from=ghcr.io/astral-sh/uv:0.4.30 /uv /uvx /bin/
+COPY --from=ghcr.io/astral-sh/uv:0.6.16 /uv /uvx /bin/
```

Comment:
Adding a new dependency (`uv add drf-standardized-errors`)  using a recent version of uv caused the revision of the lockfile to be updated to a version incompatible with uv 0.4.30. We may want to look into ways of ensuring uv version in the dev environment stays consistent with the one in the container image. For now, the versions are manually specified here and in `pyproject.toml`.

Reply:
... For every problem with python there is a tool. This tool converts the original problem into another problem. Luckily, for this problem, there is a tool...